server/src/main/java/org/elasticsearch/gateway/IncrementalClusterStateWriter.java

```diff
@@ -320,6 +358,22 @@ void rollback() {
             rollbackCleanupActions.forEach(Runnable::run);
             finished = true;
         }
+
+        void incrementIndicesWritten() {
```

Comment:
perhaps we can just make the fields package-visible and operate directly on them, removing getter and setters here, given that their are only used within `IncrementalClusterStateWriter`. Reduces the amount of clutter and avoids the extra test provisions for the interaction mocking.

Reply:
Asserting on the interactions with the `increment*` methods is deliberate; it would add a good deal more noise to check that they're being called in a different way.